t/OptionFields.t

```diff
@@ -0,0 +1,133 @@
+# This file is part of Business:PayPal:API Module.   License: Same as Perl.  See its README for details.
+# -*- mode: cperl -*-
+use Test::More;
+use strict;
+use autodie qw(:all);
+use Cwd;
+
+if ( !$ENV{WPP_TEST} || !-f $ENV{WPP_TEST} ) {
+    plan skip_all =>
+        'No WPP_TEST env var set. Please see README to run tests';
+}
+else {
+    plan tests => 14;
+}
+
+use_ok( 'Business::PayPal::API::TransactionSearch' );
+use_ok( 'Business::PayPal::API::GetTransactionDetails' );
+
+sub rndStr{ join'', @_[ map{ rand @_ } 1 .. shift ] }
+#########################
+
+require 't/API.pl';
+
+my %args = do_args();
+
+=pod
+
+These tests verify the options work.
+
+=cut
+
+my $itemName;
+
+if (-f 'options-payment.html') {
+    open(OPTIONS_PAY_HTML_READ, "<", "options-payment.html");
```

Comment:
No need to use brackets when the function is a built-in.  Don't change this if you don't feel like it, but stylistically I think it's fine to omit them.